corehq/apps/domain/models.py

```diff
@@ -51,8 +51,6 @@
 BUSINESS_UNITS = [
     "DSA",
     "DSI",
-    "DLAC",
-    "DMOZ",
```

Comment:
There should be no domains with that set (ops has been manually fixing), but I should definitely verify.  Thanks for flagging.